ethereum-bridged-carbon/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "ethereum-bridged-carbon",
+  "license": "UNLICENSED",
+  "scripts": {
+    "codegen": "graph codegen",
+    "build": "graph build",
+    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ originalpkbims/ethereum-bridged-carbon",
```

Comment:
could you update all thsee commands to be `klimadao/ethereum-bridged-carbon`? then we can just locally override that for testing on the hosted service under a diff account